packages/page-claims/src/index.tsx

```diff
@@ -73,180 +67,181 @@ const Signature = styled.textarea`
   }
 `;
 
-// FIXME Not React.FC yet
-class ClaimsApp extends TxModal<Props, State> {
-  constructor (props: Props) {
-    super(props);
-
-    this.defaultState = {
-      ...this.defaultState,
-      claim: null,
-      didCopy: false,
-      ethereumAddress: null,
-      signature: null,
-      step: 0
-    };
-    this.state = this.defaultState;
-  }
+const ClaimsApp = (props: Props) => {
+  // constructor (props: Props) {
```

Comment:
@jacogr we're refactoring this one into a FC. Is there an example of another component using TxModalNew, that we can take inspiration from?

Reply:
(shows outdated, but my question is still actual)

Reply:
That should not be a modal. I would not suggest using the TxModal, just a page with TxButton does 100%.

And yes, it should be a FC. There are 3 major areas left which has not been converted -

- claims (only done because of the testing)
- contracts (in progress)
- signer (it needs quiet before it can be done)

There are some bits and piece left in various places, so either done as a "touch this, convert" or (for the most part) "grab one or two, come back next week".

Reply:
Perfect, this simplifies things